pythran/toolchain.py

```diff
@@ -451,6 +454,8 @@ def compile_pythranfile(file_path, output_file=None, module_name=None,
 
     # Add compiled module path to search for imported modules
     sys.path.append(os.path.dirname(file_path))
+    if package is not None:
+        sys.path.append(os.getcwd())
```

Comment:
I don't get this one.

Reply:
This make the setup.py hook work nice :)

Reply:
This need improvements I know

Reply:
yeah :-)